### ISSUE TITLE
Change application from Console to Windows Application (for more stealth)

### ISSUE THREAD

**Issue by AlwaysHungrie** [exler/logey#15]
While running the code we get a flash of a console window because of the `hideConsole()` call.
It would be better if the application started without the console window in first place

**Comment by exler** [comment]
Good catch, should've been done in the first place. Mind taking this one?

**Comment by AlwaysHungrie** [comment]
I'll contribute to Readme.md
